slkspec/core.py

```diff
@@ -92,6 +93,7 @@ def __init__(
         mode: str = "rb",
         touch: bool = True,
         file_permissions: int = 0o3777,
+        delay: int = 2,
```

Comment:
If you want to make this configurable shouldn't you also add this option into the `SLKFileSystem` class?